eks-worker-al2-variables.json

```diff
@@ -33,5 +33,6 @@
     "ssh_username": "ec2-user",
     "subnet_id": "",
     "temporary_security_group_source_cidrs": "",
-    "volume_type": "gp2"
+    "volume_type": "gp2",
+    "working_dir": "{{user `remote_folder`}}/worker"
```

Comment:
Does these result in no change if you don't override these?

Reply:
This variable has to be defined, either in our variable defaults file, a user's variable file, or on the make command line.